main.c

```diff
@@ -412,6 +441,10 @@ static size_t get_codes_len(flag_type_t flag_type)
     case FLAG_TYPE_GENDERFLUID:
         return ARRAY_SIZE(codes_gfl);
         break;
+    case FLAG_TYPE_ASEXUAL:
```

Comment:
Add empty line before for consistency, also indentations and replace tabs with spaces